code/tINIT/prepHumanModelForftINIT.m

```diff
@@ -1,22 +1,37 @@
-function prepDataHumanGEM = prepHumanModelForftINIT(model, convertGenes)
+function prepDataHumanGEM = prepHumanModelForftINIT(model, convertGenes, essentialTasksFilePath, rxnsFilePath)
```

Comment:
With the more generalized scope, I would forego the `Human` word in the function name.

```suggestion
function prepDataHumanGEM = prepHumanModelForftINIT(model, convertGenes, essentialTasksFilePath, rxnsFilePath)
```

Reply:
I am also fine with leaving this as they are now, and postpone any renaming here.